plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -2982,6 +2982,13 @@ public String getVolumePath(final Connect conn, final DiskTO volume, boolean dis
         return dataPath;
     }
 
+    public static boolean useBLOCKDiskType(KVMPhysicalDisk physicalDisk) {
+        return physicalDisk != null &&
+                physicalDisk.getPool().getType() == StoragePoolType.Linstor &&
```

Comment:
@slavkap This might also interesting for you, as I think you also provide block devices to libvirt, but since QEMU 6.0. Qemu doesn't allow block devices anymore for the FILE driver.